derive/src/lib.rs

```diff
@@ -133,7 +133,7 @@ pub fn derive_anybitpattern(
 /// - All fields in the variant with discriminant 0 (if any) must implement
 ///   `Zeroable`
 ///
-/// The macro always succeeds on `union`s.
+/// The macro always succeeds on unions.
```

Comment:
Debatable change; I personally prefer to use Rust keywords whenever possible since those necessarily have a clear and unambiguous meaning.